laa_court_data_api_app/models/laa_references/internal/request/LaaReferencesPatch.py

```diff
@@ -0,0 +1,11 @@
+from typing import Optional
+from uuid import UUID
+
+from pydantic import BaseModel
+
+
+class LaaReferencesPatch(BaseModel):
+    user_name: Optional[str] = None
```

Comment:
Should these follow the changes in the prosecution cases models with `str | None = None`? Likewise for the remainder of the file, and other model files